python/bind_data_access.h

```diff
@@ -390,8 +390,12 @@ void bind_data_properties(pybind11::class_<T, Ignored...> &c) {
   c.def_property_readonly("shape",
                           [](const T &self) {
                             const auto &dims = self.dims();
-                            return std::vector<scipp::index>(
-                                dims.shape().begin(), dims.shape().end());
+                            py::list list = py::cast(dims.shape());
```

Comment:
There are some issues with Apple clang. Could this be the reason? It am not sure what `py::cast` does here? Can we use iterators or some other way to setup the list?

Reply:
Fix on https://github.com/scipp/scipp/commits/dtasev-table_html_testing or can provide a patch separately.

Reply:
Can you create a PR into this branch?